python3Packages.langchain-core: 1.0.7 -> 1.1.0; python3Packages.langchain-tests: 1.0.1 -> 1.0.2; python3Packages.anthropic: 0.71.0 -> 0.75.0; python3Packages.langchain-anthropic: 1.0.0 -> 1.2.0

sarahec opened the pull request:
1. `python3Packages.langchain-core`: bump to 1.1.0 to support anthropic API 0.75.0
2. `python3Packages.langchain-tests`: bump to 1.0.2 to pick up fixed tests (n.b. requires `blockbuster`, see https://github.com/langchain-ai/langchain/issues/34110)
3. `python3Packages.anthropic`: version bump via update script.
4. `python3Packages.langchain-anthropic`: version bump, disabled broken tests & filed upstream issues, `disabledTestPaths` -> `enabledTestPaths` per convention.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [x] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

sarahec committed:
python3Packages.anthropic: 0.71.0 -> 0.75.0

CHANGELOG: https://github.com/anthropics/anthropic-sdk-python/releases/tag/v0.75.0
DIFF: https://github.com/anthropics/anthropic-sdk-python/compare/v0.71.0...v0.75.0

sarahec committed:
python3Packages.langchain-anthropic: 1.0.0 -> 1.2.0

CHANGELOG: https://github.com/langchain-ai/langchain/releases/tag/langchain-anthropic%3D%3D1.2.0
DIFF: https://github.com/langchain-ai/langchain/compare/langchain-anthropic%3D%3D1.0.0...langchain-anthropic%3D%3D1.2.0

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 464062`
Commit: `a43d7af26dc8c4ff5d803c9571cf0d8ac7ff14f0`

---
### `aarch64-darwin`
<details>
  <summary>:x: 23 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mmengine</li>
    <li>python312Packages.mmengine.dist</li>
    <li>python312Packages.torchtune</li>
    <li>python312Packages.torchtune.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 222 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>mcp-nixos</li>
    <li>mcp-nixos.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chatlas</li>
    <li>python312Packages.chatlas.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-classic</li>
    <li>python312Packages.langchain-classic.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-mongodb</li>
    <li>python312Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-cli</li>
    <li>python312Packages.langgraph-cli.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-runtime-inmem</li>
    <li>python312Packages.langgraph-runtime-inmem.dist</li>
    <li>python312Packages.langgraph-store-mongodb</li>
    <li>python312Packages.langgraph-store-mongodb.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python313Packages.anthropic</li>
    <li>python313Packages.anthropic.dist</li>
    <li>python313Packages.chatlas</li>
    <li>python313Packages.chatlas.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.mmengine</summary>
<pre>tests/test_config/test_config.py ......................sss...            [  7%]
tests/test_config/test_lazy.py ...                                       [  8%]
tests/test_data/test_data_utils.py ..                                    [  8%]
tests/test_dataset/test_base_dataset.py ................................ [ 11%]
........                                                                 [ 12%]
tests/test_dataset/test_sampler.py .......                               [ 13%]
tests/test_device/test_device.py .                                       [ 13%]
tests/test_dist/test_dist.py ...........FFFFFFFFFFsssssssss              [ 17%]
tests/test_dist/test_utils.py .s.........FFFFFFFssssssssssssss           [ 20%]
tests/test_evaluator/test_evaluator.py ....s....                         [ 21%]
tests/test_evaluator/test_metric.py ...                                  [ 21%]
tests/test_fileio/test_backends/test_backend_utils.py .                  [ 22%]
tests/test_fileio/test_backends/test_base_storage_backend.py .           [ 22%]
tests/test_fileio/test_backends/test_lmdb_backend.py ...                 [ 22%]
tests/test_fileio/test_backends/test_local_backend.py .................. [ 24%]
.......                                                                  [ 25%]
tests/test_fileio/test_backends/test_memcached_backend.py ...            [ 25%]
tests/test_fileio/test_fileio.py .......                                 [ 26%]
tests/test_fileio/test_io.py ..............                              [ 27%]
tests/test_hooks/test_checkpoint_hook.py ...../nix/store/09c9sskp507h4kd6id9gn0yijs5y3mhd-pytest-check-hook/nix-support/setup-hook: line 23: 63209 Trace/BPT trap: 5          /nix/store/nrk1773m026hlgkk7vlpvc2s6zlwf1rh-python3-3.12.12/bin/python3.12 &quot;${flagsArray[@]}&quot;</pre>
</details>
<details>
<summary>python313Packages.mmengine</summary>
<pre>tests/test_config/test_config.py ......................sss...            [  7%]
tests/test_config/test_lazy.py ...                                       [  8%]
tests/test_data/test_data_utils.py ..                                    [  8%]
tests/test_dataset/test_base_dataset.py ................................ [ 11%]
........                                                                 [ 12%]
tests/test_dataset/test_sampler.py .......                               [ 13%]
tests/test_device/test_device.py .                                       [ 13%]
tests/test_dist/test_dist.py ...........sssssssssssssssssss              [ 17%]
tests/test_dist/test_utils.py .s.........sssssssssssssssssssss           [ 20%]
tests/test_evaluator/test_evaluator.py ....s....                         [ 21%]
tests/test_evaluator/test_metric.py ...                                  [ 21%]
tests/test_fileio/test_backends/test_backend_utils.py .                  [ 22%]
tests/test_fileio/test_backends/test_base_storage_backend.py .           [ 22%]
tests/test_fileio/test_backends/test_lmdb_backend.py ...                 [ 22%]
tests/test_fileio/test_backends/test_local_backend.py .................. [ 24%]
.......                                                                  [ 25%]
tests/test_fileio/test_backends/test_memcached_backend.py ...            [ 25%]
tests/test_fileio/test_fileio.py .......                                 [ 26%]
tests/test_fileio/test_io.py ..............                              [ 27%]
tests/test_hooks/test_checkpoint_hook.py ...../nix/store/f6pxvw5v5s6w9f59f5vz44yr6il407g3-pytest-check-hook/nix-support/setup-hook: line 23: 62420 Trace/BPT trap: 5          /nix/store/xcjk9ill54kjk8mzgq6yydnx9015lidg-python3-3.13.9/bin/python3.13 &quot;${flagsArray[@]}&quot;</pre>
</details>
</details>

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 464062`
Commit: `a43d7af26dc8c4ff5d803c9571cf0d8ac7ff14f0`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 272 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>gnome-frog</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>home-assistant-component-tests.anthropic</li>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>mcp-nixos</li>
    <li>mcp-nixos.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>newelle</li>
    <li>open-interpreter (python313Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python313Packages.open-interpreter.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chatlas</li>
    <li>python312Packages.chatlas.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.crewai</li>
    <li>python312Packages.crewai.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-classic</li>
    <li>python312Packages.langchain-classic.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-mongodb</li>
    <li>python312Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-cli</li>
    <li>python312Packages.langgraph-cli.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-runtime-inmem</li>
    <li>python312Packages.langgraph-runtime-inmem.dist</li>
    <li>python312Packages.langgraph-store-mongodb</li>
    <li>python312Packages.langgraph-store-mongodb.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mmengine</li>
    <li>python312Packages.mmengine.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.torchtune</li>
    <li>python312Packages.torchtune.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python313Packages.anthropic</li>
    <li>python313Packages.anthropic.dist</li>
    <li>python313Packages.chatlas</li>
    <li>python313Packages.chatlas.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

Erethon commented on pkgs/development/python-modules/anthropic/default.nix:
```diff
@@ -34,14 +34,14 @@
     repo = "anthropic-sdk-python";
     tag = "v${version}";
-    hash = "sha256-n8hu2RFuZN0ojHhhU4qPQaT2DUvqSglFhBo2ORln/mc=";
+    hash = "sha256-GTY4af+/eDftRog3yvxf/ImQcCDoSZVu6YPTbF41/Uk=";
```
```suggestion
    hash = "sha256-wwaH9/o7qr8eXKvmKpj+/ubOdDydcXId/qiA6dXpd2I=";
```

Erethon reviewed (commented):
I've proposed two changes because [v0.75.0 of the Anthropic SDK is out](https://github.com/anthropics/anthropic-sdk-python/releases/tag/v0.75.0).

sarahec commented on pkgs/development/python-modules/langchain-anthropic/default.nix:
```diff
@@ -43,21 +43,28 @@ buildPythonPackage rec {
-    "tests/integration_tests"
+    # TypeError from Pydantic
+    # https://github.com/langchain-ai/langchain/issues/34069
+    "tests/unit_tests/test_standard.py::TestAnthropicStandard::test_with_structured_output[PersonB] "
```
I was able to remove this because the upstream issue was resolved. Adding `python3Packages.langchain-tests @ 1.0.2` to pick up the fix.

sarahec committed:
python3Packages.langchain-core: 1.0.7 -> 1.1.0

CHANGELOG: https://github.com/langchain-ai/langchain/releases/tag/langchain-core%3D%3D1.1.0
DIFF: https://github.com/langchain-ai/langchain/compare/langchain-core%3D%3D1.0.7...langchain-core%3D%3D1.1.0

sarahec committed:
python3Packages.langchain-tests: 1.0.1 -> 1.0.2

CHANGELOG: https://github.com/langchain-ai/langchain/releases/tag/langchain-tests%3D%3D1.0.2
DIFF: https://github.com/langchain-ai/langchain/compare/langchain-tests%3D%3D1.0.1...langchain-tests%3D%3D1.0.2

dotlambda commented on pkgs/development/python-modules/langchain-tests/default.nix:
```diff
@@ -19,6 +19,7 @@
   pytest,
 
   # tests
+  blockbuster,
```
```suggestion
```

dotlambda replied:
blockbuster is not used in the langchain-tests expression, only in the langchain-anthropic expression

sarahec replied:
You're right -- it builds fine if I remove `blockbuster`. I wonder what it's doing in `uv.lock`.

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 464062`
Commit: `24ef84a6feecef72bab1ff7e9e61958227a5b104`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 272 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist</li>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>gnome-frog</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>home-assistant-component-tests.anthropic</li>
    <li>langgraph-cli (python313Packages.langgraph-cli)</li>
    <li>langgraph-cli.dist (python313Packages.langgraph-cli.dist)</li>
    <li>mcp-nixos</li>
    <li>mcp-nixos.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>newelle</li>
    <li>open-interpreter (python313Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python313Packages.open-interpreter.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.anthropic</li>
    <li>python312Packages.anthropic.dist</li>
    <li>python312Packages.chatlas</li>
    <li>python312Packages.chatlas.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.crewai</li>
    <li>python312Packages.crewai.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-classic</li>
    <li>python312Packages.langchain-classic.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-experimental</li>
    <li>python312Packages.langchain-experimental.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-google-genai</li>
    <li>python312Packages.langchain-google-genai.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-mongodb</li>
    <li>python312Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-cli</li>
    <li>python312Packages.langgraph-cli.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph-runtime-inmem</li>
    <li>python312Packages.langgraph-runtime-inmem.dist</li>
    <li>python312Packages.langgraph-store-mongodb</li>
    <li>python312Packages.langgraph-store-mongodb.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mmengine</li>
    <li>python312Packages.mmengine.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.torchtune</li>
    <li>python312Packages.torchtune.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python313Packages.anthropic</li>
    <li>python313Packages.anthropic.dist</li>
    <li>python313Packages.chatlas</li>
    <li>python313Packages.chatlas.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-classic</li>
    <li>python313Packages.langchain-classic.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-google-genai</li>
    <li>python313Packages.langchain-google-genai.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langgraph-runtime-inmem</li>
    <li>python313Packages.langgraph-runtime-inmem.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python313Packages.langgraph.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mmengine</li>
    <li>python313Packages.mmengine.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.torchtune</li>
    <li>python313Packages.torchtune.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

GaetanLepage reviewed (approved):
Diff LGTM
